cli-plugins/apps/tutorials/get-workload-status.hbs.md

```diff
@@ -0,0 +1,635 @@
+# Get Workload Status
+
+## Workload list
+
+`tanzu apps workload list` gets the workloads present in the cluster, either in the current namespace,
+in another namespace, or all namespaces.
+
+Workloads can be listed with some filters, applied through flags, such as `--namespace`, 
```

Comment:
The list result can be filtered by flags (`--namespace`, `--type` or `--app`) and `yaml` and `json` output formats are supported via the `--output` flag.

Reply:
changed